concrete_datastore/routes/views.py

```diff
@@ -103,12 +115,14 @@ def get(self, request, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        datamodel_content_json = settings.META_MODEL_DEFINITIONS
-        context['json_content'] = json.dumps(datamodel_content_json, indent=2)
+        datamodel_content_json = self._get_datamodel_format(data_format='json')
         datamodel_content_yml = self._get_datamodel_format()
-        content = DatamodelYamlToHtml(datamodel_content_json, indent=2)
+        content = DatamodelYamlToHtml(
+            settings.META_MODEL_DEFINITIONS, indent=2
+        )
         context['yaml_displayed_content'] = content.render_yaml()
```

Comment:
class is named Yaml to Html, but only function "render yaml" is called, a yaml is expected as a result.
So based on the class name "YAML to HTML", but based on parameters and function names "JSON to YAML" ?